cmake/FindADIOS2.cmake

```diff
@@ -0,0 +1,104 @@
+# - Try to find ADIOS2
```

Comment:
OK I will try to remove it although I don't know what the the alternative is.

Reply:
`adios2` creates a config file since it is built with `CMake` so, as long as you point omega_h to the correct path during the configure stage it should bring in everything it needs.